src/tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -493,6 +493,20 @@ public ApkBuilder(TaskLoggingHelper logger)
             // NOTE: we can run android-strip tool from NDK to shrink native binaries here even more.
 
             File.Copy(dynamicLib, Path.Combine(OutputDir, destRelative), true);
+
+            if (!StripDebugSymbols)
+            {
+                // symbol files need to end in .so.so so they'll show up in android studio
+                string debugLib = dynamicLib + ".dbg";
```

Comment:
Is the `.dbg` extension Windows-specific?

Reply:
No, linux specific I think.

Reply:
Right, the symbols are for the target, not host.